conf/ocsci/ocp-4.3-ga-upgrade.yaml

```diff
@@ -0,0 +1,5 @@
+---
+# Use this config for upgrading of OCP 4.3 GA cluster
+UPGRADE:
+  # ocp_upgrade_version: "4.3.3"
```

Comment:
as those config files are 4.3 related I would add here the ocs_channel like here:
https://github.com/red-hat-storage/ocs-ci/pull/1491/files#diff-4f80328bffaf5599157179f9e1a09464R119 so once we will move default config to 4.4 we will not need to change this one file.

Reply:
will move config files and fix as we discussed